lib/secureWebtask.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash');
+const Bluebird = require('bluebird');
+const Request = Bluebird.promisifyAll(require('request'));
+const Cli = require('structured-cli');
+
+const MIDDLWARE_COMPILER = '@webtask/middleware-compiler';
```

Comment:
Now that these are being used across several parts of the cli, I'd like to see these 'magic constants' centralized so that we only have to change one place to keep them up to date.

Reply:
done